library/Zend/Permissions/Rbac/Assertion/Callback.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/).
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ *
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+namespace Zend\Permissions\Rbac\Assertion;
+
+use Zend\Permissions\Rbac\AssertionInterface;
+use Zend\Permissions\Rbac\Exception\InvalidArgumentException;
+use Zend\Permissions\Rbac\Rbac;
+
+class Callback implements AssertionInterface
```

Comment:
I think `CallbackAssertion` is a better naming (it makes sense itself without considering the namespace)